example/src/main/java/io/netty/example/http/file/HttpStaticFileServerHandler.java

```diff
@@ -265,10 +265,14 @@ private static String sanitizeUri(String uri) {
     private static final Pattern ALLOWED_FILE_NAME = Pattern.compile("[A-Za-z0-9][-_A-Za-z0-9\\.]*");
 
     private static void sendListing(ChannelHandlerContext ctx, File dir) {
```

Comment:
@tomer can you just remove this one ? Its a private method + in the examples so we can "break" it